docs/advanced_examples/DecisionTreeClassifier.ipynb

```diff
@@ -332,6 +344,234 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "cv = False\n",
+    "\n",
+    "for model_name in [\"tree\", \"rf\", \"xgb\"]:\n",
```

Comment:
This is the decisiontreeclassifier notebook so not sure this should be here. However I like the visualization. We should probably have this in the documentation or in a new notebook.